womtool/src/main/scala/womtool/graph/WomGraph.scala

```diff
@@ -178,7 +178,9 @@ object WomGraph {
       clt <- CwlDecoder.decodeAllCwl(File(filePath)).
         value.
         unsafeRunSync
-      wom <- clt.womExecutable()
+      inputs = clt.requiredInputs
+      fakedInputs = JsObject(inputs map { i => i._1 -> fakeInput(i._2) })
+      wom <- clt.womExecutable(Some(fakedInputs.prettyPrint))
```

Comment:
why the fresh fakery in this PR?

Reply:
`Option`

Reply:
The faked stuff still seems like a smell to me but I don't have a better suggestion so ....